packages/adblocker/src/engine/bucket/network.ts

```diff
@@ -8,6 +8,7 @@
 
 import Config from '../../config';
 import { StaticDataView } from '../../data-view';
+import IFilter from '../../filters/interface';
```

Comment:
Nit. Because we are in a network filter bucket, we can use the `NetworkFilter` type instead of `IFilter` in this file (I think it should work).

Reply:
Solved by using existing imports: `NetworkFilter` and `CosmeticFilter`.